include/TrustWalletCore/TWCurve.h

```diff
@@ -12,11 +12,12 @@ TW_EXTERN_C_BEGIN
 /// Elliptic cruves
 TW_EXPORT_ENUM()
 enum TWCurve {
-    TWCurveSECP256k1              /* "secp256k1" */,
```

Comment:
don't change the comment format `/**/`, codegen will produce string description for swift / java